opba-protocols/xs2a-protocol-tests/xs2a-bdd-tests-common/src/main/java/de/adorsys/opba/protocol/xs2a/tests/e2e/stages/AccountInformationRequestCommon.java

```diff
@@ -458,6 +458,31 @@ public SELF user_max_musterman_selected_sca_challenge_type_email2_to_embedded_au
         return self();
     }
 
+    public SELF user_max_musterman_selected_sca_challenge_type_photo_otp_to_embedded_authorization() {
+        provideParametersToBankingProtocolWithBody(
+                AUTHORIZE_CONSENT_ENDPOINT,
+                selectedScaBody("PHOTO_OTP:photo_otp"),
+                HttpStatus.ACCEPTED
+        );
+        return self();
+    }
+
+    public SELF ui_can_read_image_data_from_obg(String user) {
+        ExtractableResponse<Response> response = withDefaultHeaders(user)
+                                                            .cookie(AUTHORIZATION_SESSION_KEY, authSessionCookie)
+                                                            .queryParam(REDIRECT_CODE_QUERY, redirectCode)
+                                                         .when()
+                                                            .get(GET_CONSENT_AUTH_STATE, serviceSessionId)
+                                                         .then()
+                                                            .statusCode(HttpStatus.OK.value())
+                                                            .extract();
+
+        assertThatResponseContainsCorrectChallengeData(response, "restrecord/tpp-ui-input/params/unknown-user-embedded-consent-challenge-data.json");
```

Comment:
Is it really unknown user or max.musterman?

Reply:
Files were renamed to max-musterman-.....